.github/workflows/test-application.yaml

```diff
@@ -248,24 +214,6 @@ jobs:
                       [mysqld]
                       default-authentication-plugin=mysql_native_password
 
-            - name: Download jackrabbit
```

Comment:
i figured that jackrabbit was not used because we have not configured it like described here: https://docs.sulu.io/en/2.2/cookbook/jackrabbit.html

decided to dop jackrabbit completely because and we dont execute anything that would use jackrabit beside of the `sulu:build` command and it would be cumbersome to configure it in the ci 😇